ext/iconv/tests/iconv_mime_encode_001.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+iconv_mime_encode(): Encoding parameter exceeds the maximum allowed for preference "input-charset"
+--CREDITS--
+Juan Carlos Morales (jcmargentina@gmail.com)
```

Comment:
The `--CREDITS--` section should no longer be used for new files, since the information is already available in the Git log.

Reply:
I will take it out